### PR TITLE
PLT-751: Enable  arrow to edit posts in comment threads

### PR DESCRIPTION
**Pull request by florianorben**

### CODE REVIEW

**Comment by it33** [comment]
Awesome!, thanks @florianorben!

File: web/react/stores/post_store.jsx [hunk 1]
```diff
@@ -239,8 +239,15 @@ class PostStoreClass extends EventEmitter {
 
         for (i; i < len; i++) {
             if (postList.posts[postList.order[i]].user_id === userId) {
-                lastPost = postList.posts[postList.order[i]];
-                break;
+                if (rootId) {
+                    if (postList.posts[postList.order[i]].root_id === rootId) {
```
Comment (hmhealey) [inline web/react/stores/post_store.jsx]: Small change that I'd add here would be to also check if the post's id is equal to rootId. As it is, you can't edit the root post of a comment thread from the RHS.

**Comment by hmhealey** [comment]
Looks good other than that one thing

**Comment by florianorben** [comment]
Fixed!

**Comment by coreyhulen** [comment]
+1

**Comment by hmhealey** [comment]
Awesome, looks good.

**Pull request merged**